nowcast/delphi_nowcast/get_epidata.py

```diff
@@ -57,37 +29,53 @@ def get_indicator_data(sensors: List[SensorConfig],
         Date that the data should be retrieved as of.
     Returns
     -------
+        Dictionary of {(source, signal, geo_type, geo_value): LocationSeries} containing indicator
+        data,
     """
     # gets all available data up to as_of day for now, could be optimized to only get a window
     output = {}
     all_combos = product(sensors, locations)
-    loop = asyncio.get_event_loop()
-    future = asyncio.ensure_future(fetch_epidata(all_combos, as_of.strftime("%Y%m%d")))
-    responses = loop.run_until_complete(future)
-    for response, sensor, location in responses:
+    all_params = [
+        {"source": "covidcast",
+         "data_source": sensor.source,
+         "signals": sensor.signal,
+         "time_type": "day",
+         "geo_type": location.geo_type,
+         "geo_value": location.geo_value,
+         "time_values": f"{EPIDATA_START_DATE}-{as_of.strftime('%Y%m%d')}",
+         "as_of": as_of.strftime("%Y%m%d")}
```

Comment:
if len(all_combos) is ever likely to be >1000, move these strftime calls outside the loop so it only gets computed once

Reply:
:+1: thanks, updated